### ISSUE TITLE
ReadMe to help first time use?

### ISSUE THREAD

**Issue by PeteBaker1** [S1ink/UE5-LidarSim#3]

**Comment by S1ink** [comment]
Yes I know. I am currently very busy and this very much a side project so possibly this summer.

**Comment by geiwp** [comment]
> Yes I know. I am currently very busy and this very much a side project so possibly this summer.

How should this plugin be used?

**Comment by S1ink** [comment]
I am currently at work so I will get back to you as soon as I can.

**Comment by geiwp** [comment]
> I am currently at work so I will get back to you as soon as I can.

OK，thanks.

**Comment by S1ink** [comment]
Ok so you will need to clone the repo and then download and extract the "third party dependencies" which I have uploaded as a release. The instructions are detailed there, although I have had many people do it wrong - the filestructure should end up as `<cloned directory>/Plugins/LidarSim/Source/ThirdParty`, with the `PCLLibrary`, `SickXDLibrary`, and `WPILibrary` directories inside. You will then need to build the project, so right click on the .uproject file to generate a Visual Studio solution file. With the solution opened, build the `UESim` project (under the "Games" folder). After that you should be able to open the .uproject in Unreal Engine. 

The project contains 2 plugins - one for visualizing point clouds, and the other for simulating a 3D lidar and filtering the point clouds using PCL. Personally I haven't used this in a couple of months so the best advice I can give right now is to analyze how everything is set up for the levels that I have in the repo. Let me know if there is anything else you are unable to figure out just by poking around.
